kubecost-cloud/kubecost-cloud-cloud-billing-integrations/kubecost-cloud-aws-integration.md

```diff
@@ -69,6 +69,6 @@ Then, navigate to the [AWS Console Cloud Formation page](https://console.aws.ama
 
 You will be prompted to provide values for several different fields to finalize your integration. See this table for working definitions of each field:
 
-<table><thead><tr><th width="218">Field</th><th>Description</th></tr></thead><tbody><tr><td>AWS Account ID</td><td>The AWS account ID where the Athena CUR is, likely your management account.</td></tr><tr><td>Master Payer ARN</td><td>Also known as the management account ARN. Configured in Step 3. The account ID of the management account where the CUR has been created.</td></tr><tr><td>Region</td><td>The AWS region Athena is running in</td></tr><tr><td>Bucket</td><td>An S3 bucket to store Athena query results that you’ve created that Kubecost has permission to access</td></tr><tr><td>Database</td><td>The name of the database created by the Athena setup</td></tr><tr><td>Table</td><td>The name of the table created by the Athena setup</td></tr><tr><td>Workgroup</td><td>Optional. Primary workgroup associated with the AWS account where your Athena CUR is.</td></tr><tr><td>Access Key Id</td><td>In the AWS IAM Console, select <em>Asset Management</em> > <em>Users</em>. Find your user and select <em>Security credentials > Create access key.</em></td></tr><tr><td>Secret Access Key</td><td>Use the Access Ket associated with the Access Key ID above.</td></tr></tbody></table>
+<table><thead><tr><th width="218">Field</th><th>Description</th></tr></thead><tbody><tr><td>AWS Account ID</td><td>The AWS account ID where the Athena CUR is, likely your management account.</td></tr><tr><td>Master Payer ARN</td><td>Also known as the management account ARN. Configured in Step 3. The account ID of the management account where the CUR has been created.</td></tr><tr><td>Region</td><td>The AWS region Athena is running in</td></tr><tr><td>Bucket</td><td>An S3 bucket to store Athena query results that you’ve created that Kubecost has permission to access. The name of the bucket should match <tt>s3://aws-athena-query-results-*</tt></td></tr><tr><td>Database</td><td>The name of the database created by the Athena setup</td></tr><tr><td>Table</td><td>The name of the table created by the Athena setup</td></tr><tr><td>Workgroup</td><td>Optional. Primary workgroup associated with the AWS account where your Athena CUR is.</td></tr><tr><td>Access Key Id</td><td>In the AWS IAM Console, select <em>Asset Management</em> > <em>Users</em>. Find your user and select <em>Security credentials > Create access key.</em></td></tr><tr><td>Secret Access Key</td><td>Use the Access Key associated with the Access Key ID above.</td></tr></tbody></table>
```

Comment:
Not sure why, but the diff is doesn't appear in this default source format. Should check it in rich format instead